lib/matplotlib/_mathtext_data.py

```diff
@@ -1032,7 +1032,54 @@
     'guilsinglleft'            : 8249,
     'plus'                     : 43,
     'thorn'                    : 254,
-    'dagger'                   : 8224
+    'dagger'                   : 8224,
+    'increment'                : 8710,
```

Comment:
I think this can be `\triangle` as well? (Which is one of the built-in latex symbols that we do not support.)

I'd suggest to add both (or find another symbol for `\triangle`?)

Reply:
The `\triangle` produces `△` and the `\increment` produces a slightly different `∆`, for which the hex value also seems to differ. 

While checking the `tex2uni` list, we have `\bigtriangleup` that uses identical Unicode as `\triangle`. Should I keep both on the list?

Reply:
OK! But there's no support for `\triangle` now, right? (At least I cannot find it in `_mathtext_data.py`, but maybe it is handled elsewhere or I am bad at searching.)

I'd say go with the "correct" one for `\triangle` and `\increment` respectively. For `\bigtriangleup` there is the question if an error is better than an almost correct symbol... I do not have a strong opinion here, but we do probably not want to remove symbols that used to work...

Reply:
Yes, that's right. There is no `\triangle` support. I have added it to the new commit.